.prettierignore

```diff
@@ -1,2 +1,4 @@
-# Styled by convention, with room for personal flair.
 README.md
+
+content/
+.forestry/
```

Comment:
This will prevent auto-formatting in these directories. This was messy before because Forestry doesn't use Prettier when they commit.